### PR TITLE
Format all json files with new script

### PR DESCRIPTION
**Pull request by ZehMatt**

### CODE REVIEW

File: objects/official/ride/openrct2.ride.alpine_coaster/object.json [hunk 1]
```diff
@@ -60,23 +40,16 @@
...
-                ]
+                "loadingPositions": [-1, 1]
             }
         ]
     },
     "strings": {
-        "name": {
-            "en-GB": "Alpine Coaster Cars"
-        },
+        "name": { "en-GB": "Alpine Coaster Cars" },
```
Comment (Gymnasiast) [inline objects/official/ride/openrct2.ride.alpine_coaster/object.json:48]: This is going to suck when inserting languages.

**Comment by Gymnasiast** [comment]
I'm not sure this is a good idea. Especially not the bit where I put a line note.

I was actually working on better scripts before exiting the team, I could finish those instead.

File: objects/official/ride/openrct2.ride.alpine_coaster/object.json (hunk 1 above)
Comment (ZehMatt) [reply objects/official/ride/openrct2.ride.alpine_coaster/object.json:48]: The languages are synchronized with a new script, the end user doesn't have to deal with this. Also JSON is not made to be human friendly at all, using JSON was the wrong choice so for consistency reasons we decided to format all the objects first with the new script to minimize diffs when synchronized.

Comment (ZehMatt) [reply objects/official/ride/openrct2.ride.alpine_coaster/object.json]: I fixed this.

**Comment by ZehMatt** [comment]
I think this is the best middle ground as for readability and density.

**Review by Gymnasiast — APPROVED** [review]
I checked a few files. It’s now much more consistent, good job :+1:


Do you plan to add some kind of CI to ensure that all files stay formatted correctly? (E.g. something like we do with `clang-format` in the main repo?)

**Comment by ZehMatt** [comment]
> I checked a few files. It’s now much more consistent, good job 👍
> 
> Do you plan to add some kind of CI to ensure that all files stay formatted correctly? (E.g. something like we do with `clang-format` in the main repo?)

That's the plan but for now I just want to get this updated so we can do the release. I want to have the scripts somewhere that the localisation repo and this one can use together, its a small collection of js modules that I wrote and since both repos share the same data formats it makes sense to have it on its own repo maybe, I'll have to think a bit about that once I get there.

**Review by Broxzier — APPROVED** [review]
This LGTM, I checked ~~all~~ some files to verify no data is lost. I'm all in for consistency.

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**